vercel-llm-api/src/vercel_ai.py

```diff
@@ -191,6 +213,25 @@ def generate(self, model_id, prompt, params={}):
         for word in new:
           yield json.loads(word)
         index = len(lines) - 1
+
+  def generate_sync(self, model_id: str, prompt: str, params={}):
```

Comment:
Why does this have to be a new function instead of just adding another argument to generate()?